catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -99,6 +99,8 @@
      */
     public static final AttributeType<Short> SHORT_TYPE;
 
+    public static final AttributeType<HashMap> HASHMAP_TYPE;
```

Comment:
Is there a reason you're creating a new type here and not just falling back to `OBJECT_TYPE`?

Regardless, using a concrete class for this definition instead of `Map` is definitely a problem.